src/test/java/org/assertj/guava/api/RangeSetAssert_containsAnyOf_Test.java

```diff
@@ -0,0 +1,288 @@
+ *
+ *    APPENDIX: How to apply the Apache License to your work.
+ *
+ *       To apply the Apache License to your work, attach the following
+ *       boilerplate notice, with the fields enclosed by brackets "[]"
+ *       replaced with your own identifying information. (Don't include
+ *       the brackets!)  The text should be enclosed in the appropriate
+ *       comment syntax for the file format. We also recommend that a
+ *       file or class name and description of purpose be included on the
+ *       same "printed page" as the copyright notice for easier
+ *       identification within third-party archives.
+ *
+ *    Copyright [yyyy] [name of copyright owner]
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.assertj.guava.api;
+
+import static com.google.common.collect.ImmutableRangeSet.of;
+import static com.google.common.collect.Range.closed;
+import static com.google.common.collect.TreeRangeSet.create;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.error.ShouldContainAnyOf.shouldContainAnyOf;
+import static org.assertj.core.internal.ErrorMessages.valuesToLookForIsEmpty;
+import static org.assertj.core.internal.ErrorMessages.valuesToLookForIsNull;
+import static org.assertj.core.util.Arrays.array;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.assertj.guava.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.common.collect.RangeSet;
+
+/**
+ * Tests for <code>{@link RangeSetAssert#containsAnyOf(Comparable[])}</code>.
+ *
+ * @author Ilya Koshaleu
+ */
+class RangeSetAssert_containsAnyOf_Test {
+
+  @Test
+  void should_fail_if_actual_is_null() {
+    // GIVEN
+    RangeSet<Integer> actual = null;
+    // WHEN
+    Throwable throwable = catchThrowable(() -> assertThat(actual).containsAnyOf(1));
+    // THEN
+    assertThat(throwable).isInstanceOf(AssertionError.class)
+                         .hasMessage(actualIsNull());
+  }
+
+  @Test
+  void should_fail_if_expected_values_are_null() {
+    // GIVEN
+    RangeSet<Integer> actual = create();
+    Integer[] expected = null;
+    // WHEN
+    Throwable throwable = catchThrowable(() -> assertThat(actual).containsAnyOf(expected));
+    // THEN
+    assertThat(throwable).isInstanceOf(IllegalArgumentException.class).hasMessage(valuesToLookForIsNull());
```

Comment:
for readability can you put chained assertions to the next line indented to the previous assertion?
I know this might be a lot work to do that everywhere! (that's the curse of big PR)

Reply:
done

Reply:
done